py/client/pydeephaven/session.py

```diff
@@ -188,15 +154,17 @@ def __init__(self, host: str = None,
         self._extra_headers = extra_headers if extra_headers else {}
 
         self.is_connected = False
+        self._auth_lock = threading.Lock()
 
         if auth_type == "Anonymous":
-            self._auth_token = auth_type
+            self._auth_value = auth_type
```

Comment:
My issue here is that this field is not the auth token.  This field is what we will place as the header value that we will send to the server in the header with an "authorization" header key.   This field normally has a value of "Bearer: XXX" where XXX is the token.  And this specifically was a source of confusion for me in the existing code where the ClientAuthHandler has a member field called `token`, which was just the token, and the Session object "auth_token", which was the whole string for the header.  So I want to have a name for this that is not misleading, because that confusion did waste me time and created pain for me before.

Reply:
I renamed the field to a completely explicit name of `_auth_header_value`, and added a comment.